data/src/main/java/com/teamagam/gimelgimel/data/message/repository/cache/room/entities/AlertFeatureEntity.java

```diff
@@ -0,0 +1,10 @@
+package com.teamagam.gimelgimel.data.message.repository.cache.room.entities;
+
+public class AlertFeatureEntity {
```

Comment:
Yeah. we could do that. However, it will not be clear what happens in the parent class. Think about a class that contains different types of entities from different packages. I wanted to be consistent about it.